### DISCUSSION TITLE
Is auto indenting supose to work well on jsx files ?

### DISCUSSION THREAD

**Discussion by luisiacc** [nvim-treesitter/nvim-treesitter#1103] [Q&A] [answered]
Is auto indenting supose to work well on jsx files ? Specifically when editing components and props.

**Comment by kyazdani42** [comment, marked as answer]
doesn't work well yet, with #1020 it should be much better

**Reply by luisiacc** [reply]
Great! The current jsx syntax from vim-jsx-pretty is kind of slow on large file with lots of code, which is natural because is regex based, so a treesitter implementation would be very nice.
